src/server/config.rs

```diff
@@ -82,6 +82,8 @@ pub struct Config {
     pub end_point_request_max_handle_duration: ReadableDuration,
     pub snap_max_write_bytes_per_sec: ReadableSize,
     pub snap_max_total_size: ReadableSize,
+    pub helper_threadpool_size: usize,
+    pub heavy_load_threshold: usize,
```

Comment:
Ditto.

Reply:
`heavy_load_threshold` is already used in `ThreadLoad`. So let's keep it.